velox/functions/lib/ArrayFunctions.h

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
It would be nice to rename this file as 'ArrayRemoveNullFunction.h' to be consistent with the class name.

Reply:
Renamed as ArrayRemoveNullFunction.h